pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java

```diff
@@ -23,15 +23,15 @@
     String upsTrackingDate;
     String upsReturnStatus;
     String upsReturnDate;
-    String HRUID;
+    String hruid;
```

Comment:
@pegahtah ok to change the name here or is it used in a response/ui somewhere?

Reply:
I think this is ok